src/newViews/AnalysisModule/Analysis/AnalysisCloneModal/styles.scss

```diff
@@ -0,0 +1,15 @@
+.modal {
+    padding: calc(var(--dui-spacing-medium) - var(--spacing-small)) !important;
```

Comment:
Why use `!important` ?

Reply:
Without it, I cannot override the padding, I tried.

Reply:
@subinasr , you should try specificity to fix this. Contact me on slack if you need more details. For now I've fixed it.